src/main/kotlin/com/coder/gateway/views/steps/CoderWorkspaceProjectIDEStepView.kt

```diff
@@ -258,9 +270,24 @@ class CoderWorkspaceProjectIDEStepView(
                                     )
                             },
                         )
+
+                    // Check the provided setting to see if there's a default IDE to set.
+                    val defaultIde = ides.find { it ->
```

Comment:
Qodana will complain about `it ->` since that is the default, although idk how much we should care about Qodana.  Possibly `ktlint` would change this automatically.

Tangent, but I think there is an alias for `find` called `firstOrNull`.  Kinda surprised there is no `indexOfFirstOrNull`, only `indexOfFirst`.  `indexOfFirst().takeIf { it >= 0 }` would work around that I guess.

Anyway, no need to change to those, more that I nerd sniped myself.  :laughing: